pegasus/sites.v3/code.org/public/hourofcode/overview.haml

```diff
@@ -3,7 +3,8 @@ title: Hour of Code Teacher Resources
 nav: hoc_teacher_resources_nav
 theme: responsive
 ---
-%link{:href=>"/css/student.css", :rel=>"stylesheet"}
+%link{href: "/css/student.css", rel: "stylesheet"}
+%link{href: "/css/tools.css", rel: "stylesheet"}
 %link(rel='stylesheet' type='text/css' href='/shared/css/course-blocks.css')
```

Comment:
Does it matter that `course-blocks.css` is linked to differently than the other two?

Reply:
would be nice to fix